docs/configuration.md

```diff
@@ -159,15 +159,24 @@ Here are the catalog properties related to locking. They are used by some catalo
 ## Hadoop configuration
 
 The following properties from the Hadoop configuration are used by the Hive Metastore connector.
-
-| Property                              | Default          | Description                                                                        |
-| ------------------------------------- | ---------------- | ---------------------------------------------------------------------------------- |
-| iceberg.hive.client-pool-size         | 5                | The size of the Hive client pool when tracking tables in HMS                       |
-| iceberg.hive.lock-timeout-ms          | 180000 (3 min)   | Maximum time in milliseconds to acquire a lock                                     |
-| iceberg.hive.lock-check-min-wait-ms   | 50               | Minimum time in milliseconds to check back on the status of lock acquisition       |
-| iceberg.hive.lock-check-max-wait-ms   | 5000             | Maximum time in milliseconds to check back on the status of lock acquisition       |
-
-Note: `iceberg.hive.lock-check-max-wait-ms` should be less than the [transaction timeout](https://cwiki.apache.org/confluence/display/Hive/Configuration+Properties#ConfigurationProperties-hive.txn.timeout) 
+The HMS table locking is a 2-step process:
+- Lock creation - the Lock itself is created and queued
+- Lock acquisition - the queued Lock should wait until every previously created Lock is released
+
+| Property                                  | Default         | Description                                                                  |
+|-------------------------------------------|-----------------|------------------------------------------------------------------------------|
+| iceberg.hive.client-pool-size             | 5               | The size of the Hive client pool when tracking tables in HMS                 |
+| iceberg.hive.lock-creation-timeout-ms     | 180000 (3 min)  | Maximum time in milliseconds to create a lock in the HMS                     |
+| iceberg.hive.lock-creation-min-wait-ms    | 50              | Minimum time in milliseconds to check the lock creation in the HMS           |
+| iceberg.hive.lock-creation-max-wait-ms    | 5000            | Maximum time in milliseconds to check the lock creation in the HMS           |
+| iceberg.hive.lock-timeout-ms              | 180000 (3 min)  | Maximum time in milliseconds to acquire a lock                               |
+| iceberg.hive.lock-check-min-wait-ms       | 50              | Minimum time in milliseconds to check back on the status of lock acquisition |
+| iceberg.hive.lock-check-max-wait-ms       | 5000            | Maximum time in milliseconds to check back on the status of lock acquisition |
```

Comment:
Same , how about, min/max time to retry checking acquisition status of lock

Reply:
Done with a somewhat reworded way - left out the "retry" from the description, and this one is an absolutely "normal" way of checking the status of the lock - no actual retry is involved